modules/peridynamics/test/tests/failure_tests/tests

```diff
@@ -26,5 +26,14 @@
     design = 'modules/peridynamics/SingularShapeTensorEliminatorUserObjectPD.md'
     issues = '#15393'
   [../]
-
+  [./2D_bond_status_convergence_HNOSPD]
+    type = 'Exodiff'
+    input = '2D_bond_status_convergence_HNOSPD.i'
+    exodiff = '2D_bond_status_convergence_HNOSPD.e'
+    map = false
+    requirement = 'BondStatusConvergedPostprocessorPD shall provide convergence criterion for PicardSolve executioner
```

Comment:
The system shall provide a means for computing the number of peridynamic bonds that changed status during the last Picard update to check convergence of the Picard iterations.